app/models.py

```diff
@@ -5,3 +5,149 @@
 Author: huxuan <i(at)huxuan.org>
 Description: Models for app.
 """
+from app import db
+
+
+class User(db.Model):
+    __tablename__ = 'userinfo'
+
+    # new segments
+    uid = db.Column(db.Integer, primary_key=True)
+    birthday = db.Column(db.DateTime)
+    student_id = db.Column(db.String(16))
+    name = db.Column(db.String(16))
+    tel = db.Column(db.String(16))
+
+    # existing segments
+    username = db.Column(db.String(32), unique=True)
+    password = db.Column(db.String(32), nullable=False)
+    # token temporarily not included
```

Comment:
I prefer to move these inline detailed comments into docstring of the model class. But you are free to leave as it is now and do it in another PR in the future. Just don't add more similiar ones.